packages/signature-controller/src/SignatureController.ts

```diff
@@ -100,20 +104,11 @@ export type SignatureControllerMessenger = RestrictedControllerMessenger<
   never
 >;
 
-export interface KeyringController {
-  signMessage: (messsageParams: MessageParams) => Promise<string>;
-  signPersonalMessage: (
-    messsageParams: PersonalMessageParams,
-  ) => Promise<string>;
-  signTypedMessage: (
-    messsageParams: TypedMessageParams,
-    options: { version: string | undefined },
-  ) => Promise<string>;
-}
-
```

Comment:
I'm a bit conflicted about this. 
We could also leave this here, accepting these types instead of taking them from `@metamask/keyring-controller`

Reply:
I think its better to take the types from `@metamask/keyring-controller`. What advantage do you see in leaving it?